### DISCUSSION TITLE
Self-Host Dependabot

### DISCUSSION THREAD

**Discussion by Cjewett** [dependabot/dependabot-core#9262] [Q&A] [answered]
Is this possible? The dependabot-script repository seems like it has been abandoned. Unfortunately we aren't able to use a Github integrated Dependabot but we wanted to investigate running it ourselves in Kubernetes. I remember trying dependabot-script a while back and was not successful, so hoping anyone has guidance on which path I should be going down to investigate this.

**Comment by yeikel** [comment, marked as answer]
Dependabot script with customizations is how I am running it

**Reply by Cjewett** [reply]
Thanks! I tried this out with Docker and it works. Had a lot of trouble trying to get all of the code building locally on Windows but I'm spinning up an Ubuntu VM to see if I run into similar issues.
